spatialos-sdk-code-generator/src/generator.rs

```diff
@@ -1,10 +1,11 @@
 use crate::schema_bundle::*;
 use heck::CamelCase;
+use std::borrow::Cow;
 use std::cell::RefCell;
 use std::collections::{BTreeMap, BTreeSet};
 use std::rc::Rc;
 
-fn get_rust_primitive_type_tag(primitive_type: &PrimitiveType) -> &str {
+fn get_rust_primitive_type_tag(primitive_type: &PrimitiveType) -> &'static str {
```

Comment:
Nit - I know you didn't add this but the name `get_rust_primitive_type_tag` is somewhat confusing

Reply:
Renamed to `primitive_type_name`, hopefully that's a bit better 😅